docs/function/handlers.md

```diff
@@ -0,0 +1,342 @@
+| How does it work?                                  | Bref turns an API Gateway event into a FastCGI (PHP-FPM) request.                                                          | Bref turns an API Gateway event into a PSR-7 request.                                 |
+| Is each request handled in a separate PHP process? | Yes (that's how PHP-FPM works).                                                                                            | Yes (Bref explicitly replicates that to avoid surprises, but that can be customized). |
+| What does the routing (i.e. separate pages)?       | Your PHP framework (one Lambda receives all the URLs).                                                                     | API Gateway: we define one Lambda and one handler class per route.                    |
+| Why does that solution exist?                      | For out-of-the-box compatibility with frameworks like Symfony and Laravel.                                                 | To match how other languages run in AWS Lambda, as recommended by AWS.                |
+| What are the use cases?                            | To build websites, APIs, etc. This should be the default approach as it's compatible with mature PHP frameworks and tools. | Build a small website, API, webhook with very little code and no framework.           |
+
+To create an HTTP handler class, Bref natively supports the [PSR-15](https://www.php-fig.org/psr/psr-15/#2-interfaces) and [PSR-7](https://www.php-fig.org/psr/psr-7/) standards:
+
+```php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+use Nyholm\Psr7\Response;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+class HttpHandler implements RequestHandlerInterface
+{
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $name = $request->getQueryParams()['name'] ?? 'world';
+
+        return new Response(200, [], "Hello $name");
+    }
+}
+
+return new HttpHandler();
+```
+
+[Full reference of HTTP events in `serverless.yml`](https://www.serverless.com/framework/docs/providers/aws/events/http-api/).
+
+## Websocket events
+
+`WebsocketHandler` instances handle Websocket events:
+
+```php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+use Bref\Context\Context;
+use Bref\Event\ApiGateway\WebsocketEvent;
+use Bref\Event\ApiGateway\WebsocketHandler;
+use Bref\Event\Http\HttpResponse;
+
+class Handler extends WebsocketHandler
+{
+    public function handleWebsocket(WebsocketEvent $event, Context $context): HttpResponse
+    {
+        $route = $event->getRouteKey();
+        $eventType = $event->getEventType();
+        $body = $event->getBody();
+
+        return new HttpResponse('ok');
```

Comment:
@danniehansen does that look correct to you? I'm adding a really small example for websockets.

Reply:
@mnapoli 👌Looks correct yes

Reply:
Thanks!